bitwarden.el

```diff
@@ -614,7 +633,7 @@ If optional argument GROUP is given, only entries in GROUP will be listed."
          :notify 'bitwarden-list-cancel-dialog
          "Cancel")
         (goto-char (point-min)))
-    (bitwarden--message "not logged in!" nil t)))
+    (bitwarden--message "vault not unlocked!" nil t)))
```

Comment:
Hmmm, what is this change for? Is it a difference between the behaviors of using the api method vs what is currently implemented? Or is this is a general change that should be for both?

Reply:
Oh, ok, I think I see. It's just better wording since the vault could be locked or logged out.